### PR TITLE
about: Add maintainers blurb

### PR DESCRIPTION
**Pull request by kallisti5**
We spoke about this on the mailing list.
https://www.freelists.org/post/haiku-commits/haiku-hrev51452-srcappsaboutsystem,2

### CODE REVIEW

File: content/about/index.md [hunk 1]
```diff
@@ -31,3 +31,20 @@ Haiku is developed by a growing community of volunteer developers from all aroun
 
   * [Discover the Haiku Community](/community)
   * Check out [Haiku Contributors](https://github.com/haiku/haiku/graphs/contributors) at GitHub
+
+### Who are the maintainers in the About window?
+
+Active developers are considered "Maintainers" of the Haiku operating system.
+
+  * Anyone who has contributed directly to the haiku or haikuporter codebase within the last 24 months.
+  * Anyone who has contributed directly to haikuports 12 or more times in the last 12 months.
```
Comment (waddlesplash) [inline content/about/index.md]: 1 contribution a month is not exactly "maintainer" ... how about we make it 32 contributions over the past 12 months? Then it's a nice power of 2.

Comment (korli) [reply content/about/index.md]: Also with commit rights.

Comment (kallisti5) [reply content/about/index.md]: I'm strictly going off of what was discussed on the mailing list. What if someone makes 1 commit of 10,000 lines?

Comment (kallisti5) [reply content/about/index.md]: Once we migrate to gerrit, the language becomes easier.  "Anyone with commit access, or who has submitted and had 16 patches accepted"?

Comment (jt15s) [reply content/about/index.md:40]: Any word on this? It's been a while (3+ years to be exact) since this was opened...

**Comment by pulkomandy** [comment]
Similar information has since been added to the "governance" page: https://www.haiku-os.org/community/organization/governance/

The version in this merge request summarizes an old discussion about when to remove people from the active maintainers list, but as far as I know this was never actually deployed. We have several people not having contributed for 3+ years and still listed as maintainers. And number of commits is not a good measure, some people don't write any code but are still members of the team for other reasons (QA testing, for example).

So the current page only says what's agreed on: how to get into the list. There is currently no clear way to say when people get removed from there.

**Comment by leavengood** [comment]
Since this has gone quite stale I am closing it. If we want to revisit it someone can reopen, but as jt15s says it has been more than 3 years so clearly there is not a lot of interest here.

I feel like we have an intuition when someone should be removed from the Maintainers list. And if one of those people comes back they can always moves themselves back. I did this in 2019 I think.

**Pull request closed without merging**